### DISCUSSION TITLE
Datatime field value is in UTC

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#1745] [Q&A] [answered]
Can we configure the timezone from setting to show the datetime accordingly in the UI? Maybe by default, show in the local timezone.

**Comment by ganigeorgiev** [comment, marked as answer]
This was requested previously but I decided not to implement it for now because it may cause some confusion when using the filters since the `date` field value in the database will be always stored as UTC and may produce unexpected results if users use a date filter in a different timezone.

There are plans in the future to normalize the date filter expressions and to allow users to use a different date format and timezone (either as zone suffix or offset) when searching, but for now it is a very low priority.
